src/server/string_family.cc

```diff
@@ -183,26 +183,27 @@ OpResult<bool> ExtendOrSkip(const OpArgs& op_args, string_view key, string_view
   return ExtendExisting(op_args, it_res->it, key, val, prepend);
 }
 
-OpResult<string> OpGet(const OpArgs& op_args, string_view key, bool del_hit = false,
-                       const DbSlice::ExpireParams& exp_params = {}) {
-  /*Get primeIterator and ExpireIterator at the same time*/
-  auto [it, it_expire] = op_args.shard->db_slice().FindExt(op_args.db_cntx, key);
+OpResult<string> OpMutableGet(const OpArgs& op_args, string_view key, bool del_hit = false,
+                              const DbSlice::ExpireParams& exp_params = {}) {
+  auto res = op_args.shard->db_slice().FindMutable(op_args.db_cntx, key);
+  res.post_updater.Run();
```

Comment:
why do you call post updater here?

Reply:
We don't update the value at any point - we either (maybe) delete it, or (maybe) expire it. Either way, no harm in calling it now.
However, now that you mention it, the call below (line 205) is redundant. I'll remove it.